go/client/chat_cli_rendering.go

```diff
@@ -169,19 +182,31 @@ func (v conversationView) show(g *libkb.GlobalContext, showDeviceName bool) erro
 	w, _ := ui.TerminalSize()
 
 	table := &flexibletable.Table{}
-	for i, m := range v.messages {
+	i := -1
```

Comment:
That's for the numbers in square brackets on the left.
It's not `i, m` because now we're skipping some messages.
```[3]    [gil 2d] oof```